mobile/test/common/integration/test_server_interface.cc

```diff
@@ -33,3 +33,10 @@ int get_server_port() {
   }
   return -1; // failure
 }
+
+void set_headers_and_data(const std::string& header_key, const std::string& header_value,
+                          const std::string& data) {
+  if (auto e = test_server()) {
```

Comment:
nit: please use readable variable names.
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

also please us the explicit type here:
https://google.github.io/styleguide/cppguide.html#Type_deduction

Under what circumstances would test_server() be false?

Reply:
I was just copying the structure of the other functions.  I think you're right though, so I removed all the test_server() checks